grippers/rr_suction_pump/src/suction_pump_topic_driver.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+Last Update 2020014, yu.okamoto@rapyuta-robotics.com
+*/
+
+#include <ros/ros.h>
+
+// #include <rr_hw_interface/gpio/libsoc_gpio.hpp>
+#include <rr_hw_interface/gpio/revpi_gpio.hpp>
+#include <rr_suction_pump/suction_pump.hpp>
+#include <rr_common/BoolMultiArray.h>
+
+namespace rapyuta
+{
+
+template <class HI, class Config>
+class SuctionPumpTopicDriver
+{
+public:
+    SuctionPumpTopicDriver(ros::NodeHandle& nh, const std::string& topic_name)
+            : _topic_name(topic_name)
+    {
+
+        int trigger_num = 0;
+        bool outputNormallyOn = false;
+        nh.getParam("pump_num", _pump_num);
+        nh.getParam("trigger_num", trigger_num);
+        nh.getParam("output_normally_on", outputNormallyOn);
+        for (int i = 0; i < _pump_num; i++) {
+            std::vector<std::string> trigger_gpio;
+            std::string trigger;
+
+            for (int j = 0; j < trigger_num; j++) {
+                nh.getParam("suction_pump" + std::to_string(i + 1) + "/trigger_gpio" + std::to_string(j), trigger);
+                trigger_gpio.push_back(trigger);
+                ROS_INFO_STREAM_NAMED(_topic_name, "trigger: " << trigger);
+            }
+            _pumps.push_back(std::unique_ptr<Pump<HI, Config>>(
+                    new Pump<HI, Config>(trigger_gpio, trigger_num, outputNormallyOn)));
+        }
+        
+        _suction_pump_cmd_sub = nh.subscribe(topic_name, 10, &SuctionPumpTopicDriver<HI, Config>::cb, this);
```

Comment:
```suggestion
        _suction_pump_cmd_sub = nh.subscribe(topic_name, 1, &SuctionPumpTopicDriver<HI, Config>::cb, this);
```
We always want the most recent message right?

Reply:
yes

Reply:
then we probably want to not allow a queue